source/Halibut/Diagnostics/HalibutTimeoutsAndLimits.cs

```diff
@@ -51,8 +51,21 @@ public class HalibutTimeoutsAndLimits
 
         /// <summary>
         ///     Amount of time to wait for a response from an RPC call.
+        /// 
+        ///     Specifically the amount of time to wait for the first byte of the request to arrive.
         /// </summary>
         public TimeSpan TcpClientReceiveResponseTimeout { get; set; } = TimeSpan.FromMinutes(10);
+        
+        /// <summary>
+        ///     Amount of time the polling service will wait for a request from an RPC call.
+        ///     Specifically the amount of time to wait for the first byte of the request to arrive.
+        ///
+        ///     This value must never be less than the PollingQueueWaitTimeout, of the client, since this
```

Comment:
If this is a hard rule, then maybe we should validate this in the 'set' of here and `PollingQueueWaitTimeout`?

Reply:
I guess we could, keep in mind that the setting on the service must be larger than the setting on the client.